modules/op-viewing-pipeline-logs-in-kibana.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+// cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc
+//
+
+[id="op-viewing-pipeline-logs-in-kibana_{context}"]
+= Viewing pipeline logs in Kibana
+
+.Procedure
+
+To view pipeline logs in the Kibana web console:
+
+. Log in to *Red Hat OpenShift Container Platform* as a cluster administrator, or a user with equivalent permissions.
```

Comment:
```suggestion
. Log in to {product-title} as a cluster administrator, or as a user with equivalent permissions.
```

Use attributes where possible for consistency with the rest of docs.
Re "or as a user with equivalent permissions" - is this possible? I would just keep it as cluster administrator but maybe there's some option in between I don't know about? I feel like mentioning this will lead users to question, what permissions do I need?